sqlglot/parser.py

```diff
@@ -1896,6 +1897,12 @@ def _parse_convert(self):
             to = None
         return self.expression(exp.Cast, this=this, to=to)
 
+    def _parse_position(self):
+        this = self._parse_term()
+        if self._match(TokenType.IN):
+            string = self._parse_term()
```

Comment:
You could do parse_bitwise here to capture more complex expressions as arguments

Reply:
Done.